.github/workflows/avm.ptn.aca-lza.hosting-environment.yml

```diff
@@ -17,7 +17,7 @@ on:
         type: boolean
         description: "Remove deployed module"
         required: false
-        default: true
+        default: false
```

Comment:
I guess that is only temporary?

Reply:
That is indeed temporary cause the workflow deployment is failing while the az cli one succeeds.

Reply:
reverted back to true.